1-mandatory/2-people-i-know.js

```diff
@@ -379,13 +389,25 @@ First, I want you to find all of my friends who are 35 or older.
 
 let thirtyFiveOrOlder = [];
 
+thirtyFiveOrOlder = friends.filter((friend) => {
+  return friend.age >= 35;
+});
+
 /*
 3) Find the email address
 Next, I want you to find all of my friends who work for "POWERNET" and then store their emails in the array below
 */
 
 let powerNetEmails = [];
 
+let worksForPowernet = friends.filter((friend) => {
```

Comment:
Nitpick: As above using function chaining and omitting braces will make this more readable